### ISSUE TITLE
Sending coverage data to coveralls from CircleCI and access token

### ISSUE THREAD

**Issue by 0xced** [SlatherOrg/slather#256]
I wasted a lot of time because I did not set the access token when trying to upload coverage data to coveralls from CircleCI.

```
Slathering...
/usr/local/lib/ruby/gems/2.3.0/gems/slather-2.3.0/lib/slather/coverage_service/coveralls.rb:193:in `post': Error while uploading coverage data to Coveralls. CI Service: circleci Message: Couldn't find a repository matching this job. (StandardError)
```

I blame coveralls for responding with an inaccurate error message (*Couldn't find a repository matching this job* instead of *Missing token*) but I think slather could help here by checking that the access token is provided.

I don’t understand why uploading from Travis to coveralls doesn’t require a token for public builds but uploading from CircleCI requires the coveralls token!

**Comment by RudyB** [comment]
@0xced Did you fix the issue?

**Comment by 0xced** [comment]
No, I just switched back to using [cpp-coveralls](https://github.com/eddyxu/cpp-coveralls) for uploading coverage information to coveralls since it handles `LCOV_EXCL_LINE` comments and slather doesn’t.
